x-pack/plugins/cross_cluster_replication/public/app/components/remote_clusters_form_field.js

```diff
@@ -0,0 +1,251 @@
+              />
+            </EuiButtonEmpty>
+          </div>
+        </Fragment>
+      );
+    };
+
+    renderNoClusterFound = () => {
+      const { intl, currentUrl } = this.props;
+      const title = intl.formatMessage({
+        id: 'xpack.crossClusterReplication.forms.emptyRemoteClustersCallOutTitle',
+        defaultMessage: 'No remote cluster found'
+      });
+
+      return (
+        <Fragment>
+          <EuiCallOut
+            title={title}
+            color="warning"
+            iconType="cross"
+          >
+            <p>
+              { this.errorMessages.noClusterFound() }
+            </p>
+
+            <EuiButton
+              {...routing.getRouterLinkProps('/add', BASE_PATH_REMOTE_CLUSTERS, { redirect: currentUrl })}
+              iconType="plusInCircle"
+              color="warning"
+            >
+              <FormattedMessage
+                id="xpack.crossClusterReplication.forms.addRemoteClusterButtonLabel"
+                defaultMessage="Add remote cluster"
+              />
+            </EuiButton>
+          </EuiCallOut>
+        </Fragment>
+      );
+    };
+
+    renderCurrentRemoteClusterNotConnected = (name) => {
+      const { intl, isEditable } = this.props;
+      const title = intl.formatMessage({
+        id: 'xpack.crossClusterReplication.forms.remoteClusterConnectionErrorTitle',
+        defaultMessage: 'Remote cluster connection error'
+      });
+
+      return (
+        <EuiCallOut
+          title={title}
+          color="warning"
+          iconType="cross"
+        >
+          <p>
+            { isEditable && this.errorMessages.remoteClusterNotConnectedEditable(name)}
+            { !isEditable && this.errorMessages.remoteClusterNotConnectedNotEditable(name)}
+          </p>
+          <EuiButton
+            {...routing.getRouterLinkProps('/', BASE_PATH_REMOTE_CLUSTERS)}
+            color="warning"
+          >
+            <FormattedMessage
+              id="xpack.crossClusterReplication.forms.viewRemoteClusterButtonLabel"
+              defaultMessage="View remote clusters"
+            />
+          </EuiButton>
+        </EuiCallOut>
+      );
+    };
+
+    renderRemoteClusterDoesNotExist = (name) => {
+      const { intl } = this.props;
+      const title = intl.formatMessage({
+        id: 'xpack.crossClusterReplication.forms.remoteClusterErrorTitle',
+        defaultMessage: 'Remote cluster error'
+      });
+
+      return (
+        <EuiCallOut
+          title={title}
+          color="warning"
+          iconType="cross"
+        >
+          <p>
+            { this.errorMessages.remoteClusterDoesNotExist(name) }
+          </p>
+          <EuiButton
+            {...routing.getRouterLinkProps('/', BASE_PATH_REMOTE_CLUSTERS)}
+            color="warning"
+          >
+            <FormattedMessage
+              id="xpack.crossClusterReplication.forms.viewRemoteClusterButtonLabel"
+              defaultMessage="View remote clusters"
```

Comment:
Instead of prompting the user to view the remote clusters, wouldn't it make more sense to prompt them to add a new one? I think we could also simplify the copy by moving some of the body text into the title.

![image](https://user-images.githubusercontent.com/1238659/51273369-59d33400-1981-11e9-866a-0bd97c6eff34.png)

Reply:
Maybe the user does not understand why suddenly the cluster does not exist anymore. This is why I thought to leave him the choice to view his clusters.

Reply:
Gotcha -- but how would the user use the list of existing clusters to figure out why a cluster no longer exists?